Hi @DubeySandeep, there seems to be a few [typescript failures](https://github.com/oppia/oppia/actions/runs/6218364051/job/16874748596?pr=18886) related to the PR.

Specifically
```
core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.spec.ts(243,45): error TS2345: Argument of type 'string' is not assignable to parameter of type 'never'.

core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.spec.ts(243,74): error TS2345: Argument of type 'string' is not assignable to parameter of type 'never'.
```

Could you please address it? Thanks!

--- a/core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.spec.ts
+++ b/core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.spec.ts
@@ -240,7 +240,7 @@ describe('Translator Overview component', () => {
   describe('when selected language is not exploration language', () => {
     beforeEach(() => {
       component.languageCode = 'hi';
-      spyOn(explorationLanguageCodeService, 'displayed').and.returnValue('en');
+      explorationLanguageCodeService.init('en');;
     });
 
     it('should show mode switcher if exploration is linked to story', () => {

--- a/core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.spec.ts
+++ b/core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.spec.ts
@@ -240,7 +240,7 @@ describe('Translator Overview component', () => {
   describe('when selected language is not exploration language', () => {
     beforeEach(() => {
       component.languageCode = 'hi';
-      explorationLanguageCodeService.init('en');;
+      explorationLanguageCodeService.init('en');
     });
 
     it('should show mode switcher if exploration is linked to story', () => {

